pytorch_lightning/logging/wandb.py

```diff
@@ -0,0 +1,116 @@
+"""
+Log using `W&B <https://www.wandb.com>`_
+
+.. code-block:: python
+
+    from pytorch_lightning.logging import WandbLogger
+    wandb_logger = WandbLogger(
+        name="my_new_nun",    # Optional, display name
+        save_dir="my_folder", # Optional, path to save data
+        offline=False,        # Optional, run offline (can sync later)
+        version="run_12345",  # Optional, used to restart previous run
+        id="run_12345",       # Optional, same as version
+        anonymous=False,      # Optional, enable or disable anonymous logging
+        project="bert",       # Optional, project to which run belongs to
+        tags=["tag1", "tag2"] # Optional, tags associated with run
+        )
+    trainer = Trainer(logger=wandb_logger)
+
+
+Use the logger anywhere in you LightningModule as follows:
+
+.. code-block:: python
+
+    def train_step(...):
+        # example
+        self.logger.experiment.whatever_wandb_supports(...)
+
+    def any_lightning_module_function_or_hook(...):
+        self.logger.experiment.whatever_wandb_supports(...)
+
+"""
+
+import os
+
+try:
+    import wandb
+except ImportError:
+    raise ImportError('Missing wandb package.')
+
+from .base import LightningLoggerBase, rank_zero_only
+
+
+class WandbLogger(LightningLoggerBase):
+    """
+    Logger for W&B.
+
+    Args:
+        name (str): display name for the run.
+        save_dir (str): path where data is saved.
+        offline (bool): run offline (data can be streamed later to wandb servers).
+        id or version (str): sets the version, mainly used to resume a previous run.
+        anonymous (bool): enables or explicitly disables anonymous logging.
+        project (str): the name of the project to which this run will belong.
+        tags (list of str): tags associated with this run.
+    """
+
+    def __init__(self, name=None, save_dir=None, offline=False, id=None, anonymous=False,
+                 version=None, project=None, tags=None):
+        super().__init__()
```

Comment:
Is it correct way? I am use to write the child class name...

Reply:
By default super will look for one class above the current class, so here it will call `LightningLoggerBase.__init__()`